webserver/web-interface/src/components/chart/StatBox.tsx

```diff
@@ -164,9 +179,10 @@ export function PressureStatBox({
       label="Pressure"
       color={theme.palette.pressure.main}
       stat={formatNumber(pressure)}
-      statTarget={formatNumber(target)}
+      statTarget={formatTarget(target)}
       unit="bar"
       sx={sx}
     />
   );
 }
+PressureStatBox.defaultProps = { sx: {} };
```

Comment:
same here

Reply:
<img width="566" alt="image" src="https://github.com/Zer0-bit/gaggiuino/assets/6816533/8a413a04-06d5-47c2-b3c9-75704d390c47">
Am I doing it wrong?

Reply:
The eslint rule is just outdated. https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/require-default-props.md

We can change it to ignore function components

Reply:
Good point! Changed it